rasa/core/processor.py

```diff
@@ -500,19 +500,22 @@ def _should_handle_message(tracker: DialogueStateTracker):
             or tracker.latest_message.intent.get("name") == USER_INTENT_RESTART
         )
 
+    def is_action_limit_reached(
+        self, num_predicted_actions: int, should_predict_another_action: bool
+    ):
```

Comment:
```suggestion
    ) -> bool:
```
(apologies, forgot about this)

Reply:
np